_docs-2/getting-started/quickstart.md

```diff
@@ -137,15 +137,19 @@ Each method above has instructions below.
 
 Start Accumulo processes (tserver, master, monitor, etc) using command below:
 
+    accumulo master
     accumulo tserver
+    accumulo monitor
```

Comment:
I think this was an example explaining how to start an individual service, generally. Rather than add the other services as an example, it would probably be better to reword the instructions to make it clear that this is just one example.

(similar comment below)

Reply:
I made some suggested improvements in #271 . Please take a look and see if they help alleviate the concerns you were trying to address in this PR.

Reply:
It looks good to me, but now I am bit biased because I read in the meantime more about the architecture of accumulo and know now a bit how to use it.

I'd highlight which of the services need to run on the local installation at the very least. I don't run for example GC and it's still fine. You may say that I should have used Uno, but maybe some other users will try their luck also by doing directly a local installation of accumulo.

Also important is a link to the wonderful accumulo-examples repository. This is a gem containing quite useful stuff to get started with accumulo.

Troubleshooting is also quite welcome. Without troubleshooting tips some of the users will stop early after trying to setup accumulo.